internal/test_helpers/fixtures/bash/history/pass

```diff
@@ -0,0 +1,122 @@
+[33m[your-program] [0m$ echo strawberry grape
+[33m[your-program] [0mstrawberry grape
+[33m[stage-2] [0m[92m✓ Command executed successfully[0m
+[33m[your-program] [0m$ echo orange mango
+[33m[your-program] [0morange mango
+[33m[stage-2] [0m[92m✓ Command executed successfully[0m
+[33m[your-program] [0m$ invalid_pineapple_command
+[33m[your-program] [0mbash: invalid_pineapple_command: command not found
+[33m[stage-2] [0m[92m✓ Received command not found message[0m
+[33m[your-program] [0m$ echo orange banana
+[33m[your-program] [0morange banana
+[33m[stage-2] [0m[92m✓ Command executed successfully[0m
+[33m[stage-2] [0m[94mPressed "<UP ARROW>" (expecting to recall "echo orange banana")[0m
+[33m[stage-2] [0m[92m✓ Prompt line matches "echo orange banana"[0m
+[33m[stage-2] [0m[94mPressed "<UP ARROW>" (expecting to recall "invalid_pineapple_command")[0m
+[33m[stage-2] [0m[92m✓ Prompt line matches "invalid_pineapple_command"[0m
+[33m[stage-2] [0m[94mPressed "<UP ARROW>" (expecting to recall "echo orange mango")[0m
+[33m[stage-2] [0m[92m✓ Prompt line matches "echo orange mango"[0m
+[33m[your-program] [0m$ echo orange mango
+[33m[stage-2] [0m[92mTest passed.[0m
+
+[33m[stage-1] [0m[94mRunning tests for Stage #1: vq0[0m
+[33m[stage-1] [0m[94mRunning ./your_shell.sh[0m
+[33m[your-program] [0m$ echo mango banana
+[33m[your-program] [0mmango banana
+[33m[stage-1] [0m[92m✓ Command executed successfully[0m
+[33m[your-program] [0m$ echo apple mango
+[33m[your-program] [0mapple mango
+[33m[stage-1] [0m[92m✓ Command executed successfully[0m
+[33m[your-program] [0m$ invalid_raspberry_command
+[33m[your-program] [0mbash: invalid_raspberry_command: command not found
+[33m[stage-1] [0m[92m✓ Received command not found message[0m
+[33m[your-program] [0m$ echo orange strawberry
+[33m[your-program] [0morange strawberry
+[33m[stage-1] [0m[92m✓ Command executed successfully[0m
+[33m[stage-1] [0m[94mPressed "<UP ARROW>" (expecting to recall "echo orange strawberry")[0m
+[33m[stage-1] [0m[92m✓ Prompt line matches "echo orange strawberry"[0m
+[33m[stage-1] [0m[94mPressed "<UP ARROW>" (expecting to recall "invalid_raspberry_command")[0m
+[33m[stage-1] [0m[92m✓ Prompt line matches "invalid_raspberry_command"[0m
+[33m[stage-1] [0m[94mPressed "<UP ARROW>" (expecting to recall "echo apple mango")[0m
+[33m[stage-1] [0m[92m✓ Prompt line matches "echo apple mango"[0m
+[33m[stage-1] [0m[94mPressed "<DOWN ARROW>" (expecting to recall "invalid_raspberry_command")[0m
+[33m[stage-1] [0m[92m✓ Prompt line matches "invalid_raspberry_command"[0m
+[33m[stage-1] [0m[94mExecuting command "invalid_raspberry_command"[0m
+[33m[your-program] [0m$ invalid_raspberry_command
+[33m[your-program] [0mbash: invalid_raspberry_command: command not found
+[33m[stage-1] [0m[92m✓ Command executed with expected error message[0m
```

Comment:
@Arpan-206 Just realized now, this seems like a good chance to split out one stage - let's split out a last stage called "Running previous commands" and test this there. The up-arrow navigation and down-arrow navigation stages can be purely about testing navigation (i.e. do the commands show in the prompt) and not actually executing them

Reply:
Can you give me a name and sutff on the slab for this @rohitpaulk?

Reply:
Want to come up with a few drafts? Will share feedback